### DISCUSSION TITLE
Feature request: New contact point type: Amazon EventBridge

### DISCUSSION THREAD

**Discussion by jangaraj** [grafana/grafana#70288] [Alerting]
Alert is a event, so it will be nice to have AWS native approach to process it as a event: Grafana puts alert (as event) to predefined Event Bus with defined IAM auth (similar as current CloudWatch datasource). Then AWS user will be able to use native AWS features to process that event.

Grafana competitors already have this feature, e.g. 
- Datadog: https://us-east-1.console.aws.amazon.com/events/home?region=us-east-1#/partners/datadoghq.com?page=overview 
- NewRelic https://us-east-1.console.aws.amazon.com/events/home?region=us-east-1#/partners/newrelic.com?page=overview
I don't think that `Partner event source` is the right implementation, but new contact point type will be sufficient.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82645. Feel free to continue the discussion around this there. Thank you!
